src/dotnet-interactive-vscode/insiders/.eslintignore

```diff
@@ -1,3 +1,3 @@
 jupyter.d.ts
-vscode.proposed.d.ts
+vscode.proposed.*.d.ts
```

Comment:
Keep both; if I understand it correctly, `vscode.proposed.d.ts` won't be caught by the pattern `vscode.proposed.*.d.ts`; there would have to be 2 dots `.` next to each other, e.g., `vscode.proposed..d.ts`.

Reply:
done! thanks for spotting the issue!